asreview/state/__init__.py

```diff
@@ -13,10 +13,11 @@
 # limitations under the License.
 
 from asreview.state.base import BaseState
-from asreview.state.dict import DictState
+from asreview.state.legacy.dict import DictState
+from asreview.state.legacy.hdf5 import HDF5StateLegacy
```

Comment:
I think we can remove all legacy state classes. This makes reviewing more straightforward.